lib/newgistics/version.rb

```diff
@@ -1,3 +1,3 @@
 module Newgistics
-  VERSION = "2.0.2"
+  VERSION = "2.0.3"
```

Comment:
I believe we are trying to follow semantic versioning, so this should be `2.3.0` because we are adding new functionality in a backwards compatible way

Reply:
Actually could we leave this change out? I think I'd rather bump the version number as a separate commit, just in case we want to add more stuff this new release

Reply:
Version bumped to 2.3.0.